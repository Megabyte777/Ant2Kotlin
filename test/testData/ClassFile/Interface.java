package testData.ClassFile;

import java.util.List;

public interface Interface {
    public void method1();
    public List<Boolean> method1(int a);
    public String method2(int a);
    public Integer method3();
}
