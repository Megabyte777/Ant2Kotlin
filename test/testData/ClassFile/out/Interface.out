method3()Ljava/lang/Integer;
method1()V
method2(I)Ljava/lang/String;
method1(I)Ljava/util/List;
